src/components/scenes/SendScene2.tsx

```diff
@@ -801,6 +801,7 @@ const SendComponent = (props: Props) => {
 
       if (onDone) {
         onDone(null, broadcastedTx)
+        navigation.pop()
```

Comment:
As mentioned at the meeting yesterday, the `navigation.pop()` should go *before* the `onDone()` call. That gives the `onDone` callback the flexibility to do things like`navigation.replace(next-step-in-the-flow)`, without the send scene's `navigation.pop` messing that up.

Reply:
👍